src/main/java/com/google/devtools/build/lib/skyframe/LocalRepositoryLookupFunction.java

```diff
@@ -91,20 +91,7 @@ public SkyValue compute(SkyKey skyKey, Environment env)
   private Optional<Boolean> maybeGetWorkspaceFileExistence(Environment env, RootedPath directory)
       throws InterruptedException, LocalRepositoryLookupFunctionException {
     try {
-      RootedPath workspaceRootedFile =
-          RootedPath.toRootedPath(
-              directory.getRoot(),
-              directory
-                  .getRootRelativePath()
-                  .getRelative(LabelConstants.WORKSPACE_DOT_BAZEL_FILE_NAME));
-      if (!workspaceRootedFile.asPath().exists()) {
-        workspaceRootedFile =
-            RootedPath.toRootedPath(
-                directory.getRoot(),
-                directory
-                    .getRootRelativePath()
-                    .getRelative(LabelConstants.WORKSPACE_FILE_NAME));
-      }
+      RootedPath workspaceRootedFile = WorkspaceFileHelper.getWorkspaceRootedFile(directory);
```

Comment:
And here, and also in other places where we deal with Skyframe.

Reply:
(Context: see my longer comment below.)

Reply:
Done.